README.md

```diff
@@ -51,6 +51,72 @@ pandas-specific things that are right now hard-coded in the theme. We also need
 to work on better configurability and extensibility. Feedback and contributions
 are very welcome!
 
+## Theme development
+
+The theme is bundled via Webpack. In `./src/*` the theme related stylesheets and javascript is located. 2 entrypoints are available:
+
+- Stylesheet: `./src/scss/index.scss'
+- Javascript: `./src/js/index.js`
+
+Both entrypoints will be bundled into `./pandas_sphinx_theme/static/{css,js}`.
+
+Theme development was inspired by the [ReadTheDocs sphinx theme](https://github.com/readthedocs/sphinx_rtd_theme).
+
+### Steps to develop the theme:
+
+1. Install yarn
+2. Install theme dependencies
+3. Run development server
+4. Build production assets
+
+**Important:** in orde to commit changes to the theme, ensure you run `yarn build:production` so all assets will be bundled into `./pandas_sphinx_theme/static/`.
+
+#### Install yarn
+
+[Yarn](https://yarnpkg.com) is a package manager we use for managing Javascript and CSS dependencies.
+
+Install via conda:
+
+```bash
+conda install yarn
+```
+
+Install alternative: https://classic.yarnpkg.com/en/docs/install.
+
+#### Install theme dependencies
+
+Install theme related dependencies:
+
+```bash
+yarn install
+```
```

Comment:
This you can run initially, but suppose you already did this, and at some point we update the dependencies (or you want to ensure you still have all up to date dependencies), what to do then? The same command?

Reply:
Indeed the same command. Yarn writes away its exact installed dependencies in the `yarn.lock` file. Even loosely defined dependencies won't mess up new installations because the lock file is your explicit reference.

Reply:
Can you mention this explicitly that if the dependencies were updated, you need to run "yarn install" again?